sirepo/job_supervisor.py

```diff
@@ -455,22 +455,22 @@ def _ops_to_cancel():
             jobCmd='compute',
             nextRequestSeconds=self.db.nextRequestSeconds,
         )
+        self.__db_init(req, prev_db=self.db)
+        self.db.pkupdate(
+            computeJobSerial=int(time.time()),
+            driverDetails=o.driver.driver_details,
```

Comment:
I think driver_details needs to get written every time, because the driver host might change.

Reply:
The instantiation of the op (which instantiates the driver) is outside of the loop so the driver details won't change.